Cargo.toml

```diff
@@ -207,29 +207,29 @@ libtock_small_panic = { path = "runtime/userspace/libtock/panic_handlers/small_p
-caliptra-auth-man-gen = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-auth-man-types = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-builder = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-emu-bus = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-emu-cpu = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-emu-derive = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-emu-periph = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-emu-types = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-error = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e", default-features = false }
-caliptra-hw-model = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-hw-model-types = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-image-crypto = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e", default-features = false, features = ["rustcrypto"] }
-caliptra-image-fake-keys = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-image-gen = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-image-types = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-registers = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-test = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-test-harness = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-caliptra-test-harness-types = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-ureg = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e" }
-dpe = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "2b33b0dde6da3ef10666d813d1a874716ebf652e", default-features = false, features = ["dpe_profile_p384_sha384"] }
+caliptra-api = { git = "https://github.com/chipsalliance/caliptra-sw", rev = "b73ced96611a4a1ae7823d9b4261e4f467bc0279" }
```

Comment:
We will need to update these commit SHAs before merging, as they will probably change when https://github.com/chipsalliance/caliptra-sw/pull/2513 is merged